### PR TITLE
Cleanup some Shortcuts

### PR DESCRIPTION
**Pull request by ZeldaZach**
Some shortcuts were missing, and others were mislabeled. I corrected the ones relating to Top, Bottom, and Selected cards.

Screenshot 2020-04-30 17 04 38

### CODE REVIEW

**Comment by ebbit1q** [comment]
The capitalization on the shortcuts is now less consistent, the other shortcuts only have their first letter capitalized, not every noun.

File: cockatrice/src/shortcutssettings.h [hunk 1]
```diff
@@ -473,24 +476,34 @@ class ShortcutsSettings : public QObject
         {"Player/aViewSideboard", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Sideboard"),
                                               parseSequenceString("Ctrl+F3"),
                                               ShortcutGroup::View)},
-        {"Player/aViewTopCards", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Top cards of library"),
+        {"Player/aViewTopCards", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Top Card(s) of Library"),
```
Comment (ebbit1q) [inline cockatrice/src/shortcutssettings.h:479]: The function is called "View top cards of library..." in the right click menu, we use (s) in cases where the s gets added depending on a variable.

Comment (ZeldaZach) [reply cockatrice/src/shortcutssettings.h:479]: Yeah my B

**Comment by ZeldaZach** [comment]
Re-addressed and ensured proper capitalization across the board

File: cockatrice/src/shortcutssettings.h [hunk 2]
```diff
@@ -504,18 +505,18 @@ class ShortcutsSettings : public QObject
...
+        {"Player/aLeaveGame", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Leave Game"),
                                           parseSequenceString("Ctrl+Q"),
                                           ShortcutGroup::Gameplay)},
         {"Player/aConcede",
          ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Concede"), parseSequenceString("F2"), ShortcutGroup::Gameplay)},
-        {"Player/aRollDie", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Roll dice"),
+        {"Player/aRollDie", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Roll Dice"),
                                         parseSequenceString("Ctrl+I"),
                                         ShortcutGroup::Gameplay)},
         {"Player/aShuffle", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Shuffle library"),
```
Comment (tooomm) [inline cockatrice/src/shortcutssettings.h:522]: `Shuffle Library`

File: cockatrice/src/shortcutssettings.h [hunk 3]
```diff
@@ -157,214 +160,215 @@ class ShortcutsSettings : public QObject
...
-        {"Player/aIncPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Add power and toughness (+1/+1)"),
+        {"Player/aIncPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Add Power and Toughness (+1/+1)"),
                                       parseSequenceString("Ctrl+Alt++"),
                                       ShortcutGroup::Power_Toughness)},
-        {"Player/aDecPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Remove power and toughness (-1/-1)"),
+        {"Player/aDecPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Remove Power and Toughness (-1/-1)"),
                                       parseSequenceString("Ctrl+Alt+-"),
                                       ShortcutGroup::Power_Toughness)},
-        {"Player/aSetPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Set power and toughness"),
+        {"Player/aSetPT", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Set Power and Toughness"),
```
Comment (tooomm) [inline cockatrice/src/shortcutssettings.h:368]: `Set Power and Toughness...`

Comment (ZeldaZach) [reply cockatrice/src/shortcutssettings.h:368]: I'll add the dots now, but I'm not calling it a blocker to test them all at the moment

Comment (tooomm) [reply cockatrice/src/shortcutssettings.h]: Collected and connected all the dots... :)

**Comment by tooomm** [comment]
Please double check my changes before merging.

Grouping/Categories are still not ideal and I it looks like they follow different philosophies.

**Comment by ZeldaZach** [comment]
We can address grouping in a future issue, I'd rather not hold this back.

**Pull request merged**